paddle/framework/op_kernel.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+
```

Comment:
Very bad code here

Since the `op_kernel.h` cannot be include by any order.

Reply:
rm op_kernel.h since the kernel type will be moved to op_kernel_type.h

Reply:
fixed, rm op_kernel.h since it will be refined to op_kernel_type.h